tools/orca/src/models/basic.rs

```diff
@@ -61,28 +67,43 @@ impl ActorModel for ActorBasic {
 
 impl ActorBasic {
     fn next_transition(&mut self, roles: &BTreeSet<ActorRole>) -> Transition {
+        let logout_transition = Transition {
+            delay: Some(Duration::from_secs(5)),
+            action: TransitionAction::Logout,
+        };
         match self.state {
             State::Unauthenticated => Transition {
                 delay: None,
                 action: TransitionAction::Login,
             },
+            // Doing some tests with more people I noticed that if the delay is too low somehow??! the server could start processing the reauth request before
+            // the auth one, yielding an error,
+            // TODO!!: understand why that happens
```

Comment:
re-auth can't proceed unless the session is stable in the DB. Session data is written asynchronously (It's in a queue). It's likely that you're hitting this "so fast" that the session hasn't drained out the queue yet. 

It may be worth opening an issue to find some way to batch or aggregate session write back to improve it's performance. But that's a future thing.